### PR TITLE
Fix authenticate, remove app bridge from auth full redirect page

### PR DESCRIPTION
**Pull request by enmaboya**
The problem was that App Bridge redirected the user back to Shopify before the JavaScript script had a chance to run. 

In theory, after being redirected back to Shopify, our script should have executed, but since we were not yet authenticated at that moment, an error occurred.

### CODE REVIEW

File: src/resources/views/auth/fullpage_redirect.blade.php [hunk 1]
```diff
@@ -4,7 +4,6 @@
         <meta charset="utf-8">
         <base target="_top">
         <meta name="shopify-api-key" content="{{ \Osiset\ShopifyApp\Util::getShopifyConfig('api_key', $shopDomain ?? Auth::user()->name ) }}"/>
-        <script src="https://cdn.shopify.com/shopifycloud/app-bridge.js"></script>
 
         <title>Redirecting...</title>
```
Comment (jsalguero94) [inline src/resources/views/auth/fullpage_redirect.blade.php:9]: why do we need this view? it works if we make the redirect from the controller without using the view.

Comment (enmaboya) [reply src/resources/views/auth/fullpage_redirect.blade.php:9]: As I mentioned in the issue, there is no difference; both options will likely work the same way.

However, someone might be using this view to send analytics via sendBeacon or for something else. I can't make any definitive statements here.

Personally, I don't use the authenticate route at all—it's not necessary during a token exchange.

Comment (Kyon147) [reply src/resources/views/auth/fullpage_redirect.blade.php:9]: This is from memory but...

Full page redirect view is for after the /authenticate route runs, this full page redirect is what moves you to to auth screen on Shopify to accept the install.

**Review by Kyon147 — APPROVED** [review]

**Comment by gofortiss** [comment]
You are right this view is usless

**Pull request merged**